velox/connectors/hive/storage_adapters/abfs/tests/AbfsFileSystemTest.cpp

```diff
@@ -72,13 +75,53 @@ class AbfsFileSystemTest : public testing::Test {
     azuriteServer->stop();
   }
 
+  std::unique_ptr<WriteFile> openFileForWrite(
+      std::string_view path,
+      std::shared_ptr<
+          facebook::velox::filesystems::test::MockBlobStorageFileClient>
+          client) {
+    auto abfsfile =
+        std::make_unique<facebook::velox::filesystems::abfs::AbfsWriteFile>(
+            std::string(path), azuriteServer->connectionStr());
+    abfsfile->testingSetFileClient(client);
+    abfsfile->initialize();
+    return abfsfile;
+  }
+
+  static std::string generateRandomData(int size) {
```

Comment:
@majetideepak this method giving a flexibility to generate any size data. I think generating ramdom content is fine, since we can keep a duplicate one in mem and veiry with data got from actually read.